Makefile

```diff
@@ -0,0 +1,19 @@
+src = $(wildcard *.c)
+obj = $(src:.c=.o)
+
+LDFLAGS = -lcurl -lssl -lcrypto
+DEBUG = -g -Wall -Wextra
+
+sentiment: $(obj)
+	    $(CC) -o $@ $^ $(LDFLAGS)
```

Comment:
Nevermind, I found that it should be defined in the shell.

Reply:
Oh... I forgot to define the CC at the Makefile =p, and you are right, is a good practice to define the preferred compiler on the Makefile in case the variable isnt defined in OS

Reply:
Yes, but doing so will overwrite the CC defined by the user's shell, right?

Reply:
Using the "?=" will only define CC if the user's shell variable isn't defined, so its more "safe"

Reply:
Oh ok, that's a nice feature.